### PR TITLE
Win installer: make it possible to unselect desktop Shortcut 

### PR DESCRIPTION
**Pull request by ann0see**
This should also fix the $PWD issue

### CODE REVIEW

File: windows/installer.nsi [hunk 1]
```diff
@@ -281,7 +288,11 @@ Section "un.Install"
     ${EndIf}
 
     ; Remove the Start Menu and desktop shortcuts
-    Delete   "$DESKTOP\${APP_NAME}.lnk"
+    IfFileExists "$DESKTOP\${APP_NAME}.lnk" deletedtshortcut continueremove
+    deletedtshortcut:
```
Comment (pljones) [inline windows/installer.nsi:292]: `deletedtshortcut` -- should be `deletetshortcut` (i.e. not past tense).

File: windows/installer.nsi [hunk 2]
```diff
@@ -281,7 +288,11 @@ Section "un.Install"
     ${EndIf}
 
     ; Remove the Start Menu and desktop shortcuts
-    Delete   "$DESKTOP\${APP_NAME}.lnk"
+    IfFileExists "$DESKTOP\${APP_NAME}.lnk" deletedtshortcut continueremove
```
Comment (pljones) [inline windows/installer.nsi:291]: The `deletedtshortcut` is visually hard to parse...  Perhaps just `deleteshortcut`.  Also, I'd prefer `continueremove` to be `skipshortcut`, as it's the "else" to this "if".

Comment (ann0see) [reply windows/installer.nsi]: Should be done now
